scripts/build-isa-sim.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
```

Comment:
Is this used anywhere? I think we should avoid having duplicate scripts so build-toolchains or whatever should use this unless there is something else this is doing.

Reply:
i use it all the time. if more people find it preferable, i can add an option to `./scripts/build-toolchains.sh -isa-sim-only`, but that seems more error prone than this approach. 

these are all small one-off convenience scripts, so i personally think the code duplication across them is not as big of deal as making a script more complicated

Reply:
The quicker solution for incremental builds is just to navigate to the riscv-isa-sim build directory and rerun `make && make install`.

While useful, all this seems to be outside of the scope of this PR.  I would also prefer that the individual components be refactored into different shell functions in `build-toolchains.sh`, which would avoid a lot of the common boilerplate with separate scripts.

Reply:
i'll remove it from this pr and add it to another branch then